pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -158,6 +158,13 @@ func isARM64(arch string) bool {
 	return false
 }
 
+func isS390X(arch string) bool {
+	if arch == "s390x" {
+		return true
+	}
+	return false
+}
+
```

Comment:
This could be

```golang
func isS390X(arch string) bool {
    return arch == "s390x"
}
```

and the existing arch-specific functions could be refactored too.

Reply:
Yes, i must have missed this one when i changed the other ones...

Reply:
Fixed, and this time i triple checked that i didn't miss any other instances.